src/android/Geolocation.java

```diff
@@ -94,6 +94,12 @@ private boolean hasPermission() {
 
     @Override
     public void onRequestPermissionResult(int requestCode, String[] permissions, int[] grantResults) {
+        // In case a permission request is cancelled, the permissions and grantResults arrays are empty.
```

Comment:
I'm unsure if this is true for multiple permission prompts, but since it's only 1 it should be fine.